actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -50,6 +50,9 @@ module Redirecting
     #   redirect_to post_url(@post), status: 301, flash: { updated_post_id: @post.id }
     #   redirect_to({ action: 'atom' }, alert: "Something serious happened")
     #
+    # Statements after redirect_to in our controller gets executed, so redirect_to doesn't stops the execution of the function.
```

Comment:
"Statements after redirect_to in our controller get executed, so redirect_to doesn't stop the execution of the function."

Some small edits.